spec/footer_spec.rb

```diff
@@ -1,4 +1,5 @@
 require_relative '../app/pages/page methods/mega-menu_met'
+require_relative '../app/assertions/footer_assert'
 
 
 describe '', :LEAGUE76 do
```

Comment:
Add the name of regression suite between the single quotes. In this case, the regression suite name is _Footer_